vyper/compiler/input_bundle.py

```diff
@@ -0,0 +1,88 @@
+from dataclasses import dataclass
+from typing import Any
+
+from pathlib import Path, PurePath
+
+
+# stub
+class CompilerInput:
```

Comment:
Maybe `ImportedFile`

Reply:
it's any input file, including a compilation target